seeder/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Inventory struct {
+	db *sql.DB
+}
+
+const file string = "products.db"
+
+func newDB() {
+	db, err := os.Create(file)
+	if err != nil {
+		fmt.Print("Failed to creat a new database")
+		return
+	}
+	defer db.Close()
+}
+
+const productTable string = `
+  CREATE TABLE IF NOT EXISTS products (
+  "id" INTEGER PRIMARY KEY AUTOINCREMENT,
+  "name" TEXT,
+  "price" REAL, 
+  );`
+
+const jsonFile string = "products.json"
+
+type ProductStruct struct {
+	Product []Product
+}
+
+type Product struct {
+	ID    int    `json:"id"`
+	Name  string `json:"name"`
+	Price int    `json:"price"`
+}
+
+func main() {
+	err := os.Remove(file)
+	if err != nil {
+		fmt.Println("Failed to delete existing database")
+	}
+
+	data, err := os.Open(jsonFile)
+	if err != nil {
+		fmt.Println("Failed to open json file")
+		return
+	}
+
+	fmt.Println("File opened")
+
+	defer data.Close()
+
+	byteResult, err := os.ReadFile(jsonFile)
+	if err != nil {
+		fmt.Println("Failed to read json file")
+		return
+	}
+
+	list := ProductStruct{}
+	if err := json.Unmarshal(byteResult, &list.Product); err != nil {
+		fmt.Printf("Failed to unmarshal json file")
+		return
+	}
+
+	db, _ := sql.Open("sqlite3", file)
+
+	newDB()
+	createTable(db)
+
+	var newItem []Product
+
+	for i := 0; i < 5; i++ {
```

Comment:
Let the list of products you obtain from the json file drive the number of iterations. No hard-coding.